src/pages/reference/handlers/soap.md

```diff
@@ -0,0 +1,91 @@
+
+- `source`(type: `String`, required) - A URL to your WSDL, must be identical to `wsdl`
+- `wsdl` (type: `String`, required) - A URL to your WSDL, must be identical to the `source`
+- `basicAuth` (type: `Object`) - Basic Authentication Configuration, includes the following fields:
+  - `username` (type: `String`, required) - Username for Basic Authentication
+  - `password` (type: `String`, required) - Password for Basic Authentication
+- `securityCert` (type: `Object`) - SSL Certificate Based Authentication Configuration, includes the following fields:
+  - `publicKey` (type: `String`) - Public key
+  - `privateKey` (type: `String`) - Private key
+  - `password` (type: `String`) - Password
+  - `publicKeyPath` (type: `String`) - Path to the file or URL that contains your public key
+  - `privateKeyPath` (type: `String`) - Path to the file or URL that contains your private key
+  - `passwordPath` (type: `String`) - Path to the file or URL that contains your password
+-  `schemaHeaders` (type: `Any`) - JSON object for adding headers to API calls for runtime schema introspection
+   - You can also provide `.js` or `.ts` file path that exports `schemaHeaders` as an object.
+-  `operationHeaders` (type: `JSON`) - JSON object for adding headers to API calls for runtime operation execution
+- `includePorts` (type: `Boolean`) - A flag that defines if ports defined in the WSDL are represented in the schema
+  - If `true`, the ports defined in the WSDL are represented as GraphQL-type objects in the schema. The fields of the object will be the operations of the port. Most SOAP endpoints only define one port, so including it in the schema will be inconvenient, but if there are multiple ports with operations of the same name, you should set this option to `true`. If set to `false` (default), only one of the identically-named operations will be callable.
+- `includeServices` (type: `Boolean`) - A flag that defines if services defined in the WSDL are represented in the schema
+  - If `true`, the services defined in the WSDL will be represented as GraphQL-type objects in the schema. The fields of the object will be the ports of the service (or the operation, dependent on `includePorts`). Most SOAP endpoints only define one service, so including it in the schema will be inconvenient, but if there are multiple services with operations of the same name, you should set this option to `true`. If set to `false` (default), only one of the identically-named operations will be callable.
+- `selectQueryOrMutationField` (type: `Array of Object`) - Allows you to explicitly override the default operation (Query or Mutation) for any SOAP operation:
+  - `service` (type: `String`, required)
+  - `port` (type: `String`, required)
+  - `operation` (type: `String`, required)
+  - `type` (type: `String (query | mutation)`, required)
+- `selectQueryOperationsAuto` (type: `Boolean`) - Automatically adds operations starting with `query` or `get` to the Query type
```

Comment:
```suggestion
- `source`(type: `String`, required) - A URL to your WSDL, must be identical to `wsdl`
- `wsdl` (type: `String`, required) - A URL to your WSDL, must be identical to the `source`
-  `schemaHeaders` (type: `Any`) - JSON object for adding headers to API calls for runtime schema introspection
   - You can also provide `.js` or `.ts` file path that exports `schemaHeaders` as an object.
-  `operationHeaders` (type: `JSON`) - JSON object for adding headers to API calls for runtime operation execution
```

We only support `source, wsdl, operationHeaders and schemaHeaders` config options. Verified by checking the [types on the PR](https://git.corp.adobe.com/commerce-gateway/utils/pull/26/files).

![image](https://github.com/AdobeDocs/graphql-mesh-gateway/assets/35203638/e97ad8b5-40d2-48a1-8eb6-f323b23b31d8)